RELEASES.md

```diff
@@ -1,79 +1,140 @@
-Version 1.0.0-beta (April 2015)
--------------------------------------
-
-* ~1100 changes, numerous bugfixes
```

Comment:
Thanks for mentioning me!

This is me:
Heejong Ahn <heejongahn@gmail.com <poga.bahamut@gmail.com>>

p.s) To be frank, it's my first (and misc) open-source contribution and I
don't know what's really going on here so I just mocked what Poga's mail..
Plz let me know if I've done anything wrong.

2015-05-11 14:28 GMT+09:00 Tshepang Lekhonkhobe <notifications@github.com>:

> In RELEASES.md
> <https://github.com/rust-lang/rust/pull/25196#discussion_r30013677>:
>
> > @@ -1,79 +1,140 @@
> > -Version 1.0.0-beta (April 2015)
> > --------------------------------------
> > -
> > -* ~1100 changes, numerous bugfixes
>
> am curious why remove change count
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rust-lang/rust/pull/25196/files#r30013677>.
>